napalm_logs/config/ios/INTERFACE_ADMIN_DOWN.yml

```diff
@@ -0,0 +1,14 @@
+# Interface GigabitEthernet2, changed state to administratively down
+
+messages:
+  - error: INTERFACE_ADMIN_DOWN
```

Comment:
I think it would make sense to just leave it as `INTERFACE_DOWN`, as there's a dedicated field `admin_status` vs. `oper_status`.

Reply:
I hear you, and I understand it would be more straight forward, however this is not a strong enough reason to introduce another layer of explicit name (i.e., we could also have had `INTERFACE_OPER_DOWN`) - let's group them as much as possible, as it's easy to match on `INTERFACE_DOWN` then look for the corresponding flag. Speaking of which, a message with the following structure under the `INTERFACE_OPER_DOWN` would be redundant, as the oper status is already told pointed as being down from both message name and body without any real beneift:

```json
{
  "yang_message": {
    "interfaces": {
      "interface": {
        "GigabitEthernet2": {
          "state": {
            "oper_status": "DOWN"
          }
        }
      }
    }
  }
}
```

Thoughts?

Reply:
Changed it so that admin and oper status use `INTERFACE_DOWN`